include/pagmo/population.hpp

```diff
@@ -88,7 +88,7 @@ class population
      * Constructs an empty population with a pagmo::null_problem.
      * The random seed is initialised to zero.
      */
-    population() : m_prob(null_problem{}), m_e(0u), m_seed(0u)
+    population() : m_prob(null_problem{}), m_ID(), m_x(), m_f(), m_champion_x(), m_champion_f(), m_e(0u), m_seed(0u)
```

Comment:
I just checked on a local branch, and they did not go away.

I ended up with 2 ``population_test_case`` tests in ``test.py`` as well.